### PR TITLE
MAJOR: Bump Node 10

### PR DESCRIPTION
**Pull request by snewcomer**
ref #267 

I'm not entirely sure if just testem bump or package.json bump constitutes a major release, but this PR will include:

- Bumping package.json node to minimum 10 and bumping dependencies
- Adding eslint and resolving some errors

After this is merged, follow up for #281

### CODE REVIEW

**Comment by snewcomer** [comment]
@alexBaizeau This PR is ready for your review!

File: addon/services/media.js [hunk 1]
```diff
@@ -87,7 +88,7 @@ export default Service.extend(Evented, {
   * @type      Ember.NativeArray
   * @default   Ember.NativeArray
   */
-  matches: computed(function() {
+  matches: computed('_mocked', '_mockedBreakpoint', function() {
```
Comment (alexBaizeau) [inline addon/services/media.js:91]: 👍

File: tests/dummy/config/targets.js [hunk 2]
```diff
@@ -1,8 +1,18 @@
...
+const browsers = [
+  'last 1 Chrome versions',
+  'last 1 Firefox versions',
+  'last 1 Safari versions'
+];
+
+const isCI = !!process.env.CI;
+const isProduction = process.env.EMBER_ENV === 'production';
+
+if (isCI || isProduction) {
```
Comment (alexBaizeau) [inline tests/dummy/config/targets.js:12]: 👍

**Comment by alexBaizeau** [comment]
This looks pretty good to me! We'll do a major bump release when https://github.com/freshbooks/ember-responsive/issues/281 is addressed.

**Pull request merged**